lib/sensors/inverter_current.cpp

```diff
@@ -0,0 +1,38 @@
+#include <inverter_current.hpp>
+#include "adc_mux.hpp"
+
+namespace hyped::sensors {
+
+std::optional<InverterCurrent> InverterCurrent::create(core::ILogger &logger,
```

Comment:
you don't need a create method unless there is stuff that can cause the constructor to fail